kotrcli.go

```diff
@@ -15,30 +17,47 @@ const (
 	TAIKIN
 )
 
-func Dakoku(val Type, userToken string, token string) (string, error) {
-
+func Dakoku(val Type, userToken string, token string) error {
 	client := &http.Client{}
 	t := time.Now()
-	unique_timestamp := t.Format("20060102150405")
-	param := fmt.Sprintf("id=%v&highAccuracyFlg=false&credential_code=40&user_token=%v&comment=&unique_timestamp=%v&version=1.2.7&token=%v", getTypeValue(val), userToken, unique_timestamp, token)
-	buffer := bytes.NewBufferString(param)
+	uniqueTimestamp := t.Format("20060102150405")
 
-	request, err := http.NewRequest("POST", "https://s2.kingtime.jp/gateway/bprgateway", buffer)
+	param := url.Values{}
+	param.Add("id", getTypeValue(val))
+	param.Add("highAccuracyFlg", "false")
+	param.Add("credential_code", "40")
+	param.Add("user_token", userToken)
+	param.Add("unique_timestamp", uniqueTimestamp)
+	param.Add("comment", "")
+	param.Add("version", "1.2.7")
+	param.Add("token", token)
+	request, err := http.NewRequest("POST", "https://s2.kingtime.jp/gateway/bprgateway", bytes.NewBufferString(param.Encode()))
 	request.Header.Add("Content-Type", "application/x-www-form-urlencoded; charset=UTF-8")
 	request.Header.Add("Accept", "application/json, text/javascript, */*; q=0.01")
 	request.Header.Add("X-Requested-With", "XMLHttpRequest")
 	request.Header.Add("Connection", "keep-alive")
 
 	resp, err := client.Do(request)
 	if err != nil {
-		return "", err
+		return err
 	}
 	defer resp.Body.Close()
+	if resp.StatusCode < 400 {
+		switch val {
+		case SYUKKIN:
+			fmt.Println("Let's enjoy at work")
+		case TAIKIN:
+			fmt.Println("See you next time")
+		default:
+			fmt.Println("fmm... what's happen?")
+		}
+	}
 	body, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
-		return "", err
+		return err
 	}
-	return string(body), nil
+	fmt.Fprintln(os.Stderr, string(body))
+	return nil
```

Comment:
thanks prrrrr!!!!! @wreulicke 
I consider about integration with AWS Lambda via HTTP for slack command.
When I create Lambda Function, I suppose to use this client as library so I think caller need kot's response body for lambda http response(for slack).
What should we do for this integration?

Reply:
fmm...
OK, We should use io.Writer such as bytes.Buffer.

Reply:
We can create entrypoint like `lambda/main.go` if we need executable for lambda.

example using bytes.Buffer 

```go
func Handler(.....) {
  buf := new(bytes.Buffer)
  _ := kotrcli.Dakoku(kotrcli.Syukkin, userToken, token)
  body := buf.String()
  // .............
}
```